packages/backend/src/api/endpoints/SessionApi.ts

```diff
@@ -55,4 +64,13 @@ export class SessionAPI extends AbstractAPI {
       path: joinPaths(basePath, sessionId, 'tokens', template || ''),
     });
   }
+
+  public async refreshSession(sessionId: string, params: RefreshTokenParams) {
+    this.requireId(sessionId);
+    return this.request<Cookies>({
+      method: 'POST',
+      path: joinPaths(basePath, sessionId, 'tokens'),
```

Comment:
I think this is got renamed to `refresh`:

```suggestion
      path: joinPaths(basePath, sessionId, 'refresh'),
```